mhs/outbound/outbound/request/synchronous/tests/test_handler.py

```diff
@@ -47,7 +47,7 @@ def test_post_message(self, mock_message_id, mock_correlation_id, mock_get_uuid)
 
         self.config_manager.get_interaction_details.assert_called_with(INTERACTION_NAME)
         self.workflow.handle_outbound_message.assert_called_with(MOCK_UUID, MOCK_UUID_2, INTERACTION_DETAILS,
-                                                                 REQUEST_BODY)
```

Comment:
You've added several bits of logic to this handler, so I'd expect to see additional test cases for them.

Reply:
Added test cases to ensure the state is being updated appropriately based on whether or not the response was sent to the supplier system